docs/docs/recipes/custom-domain/README.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 9
+---
+
+# 🌍 Custom domain
+
+_Added in v1.6.0_
```

Comment:
i think no need for this line since it's cloud?

Reply:
same for other pages

Reply:
all removed